modules/ztp-configuring-hwevents-using-pgt.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-creating-hwevents_{context}"]
+= Configuring bare-metal events monitoring using PolicyGenTemplate CRs
+
+You can configure bare-metal hardware events for vRAN clusters that are deployed using the GitOps Zero Touch Provisioning (ZTP) pipeline.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the {redfish-operator}.
+* Create a `BMCEventSubscription` CR for the BMC Redfish hardware.
+* Create a Git repository where you manage your custom site configuration data.
+
+[NOTE]
+====
+Multiple `HardwareEvent` resources are not permitted.
+====
+
+.Procedure
+
+. Create the `HardwareEvent` custom resource (CR) entry in the `PolicyGenTemplate`:
+
+.. Save the following YAML in the `hw-event.yaml` file:
++
+[source,yaml]
+----
+apiVersion: event.redhat-cne.org/v1alpha1
+kind: HardwareEvent
+metadata:
+  name: openshift-hw-events
+  namespace: openshift-hw-events
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  nodeSelector: {}
+  transportHost: "amqp://amq-router.amq-router.svc.cluster.local"
+  logLevel: "trace"
+----
+
+.. Create the `HardwareEvent` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f hw-event.yaml
```

Comment:
I don't think the above steps are needed since the yaml is already created in the ztp repo https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/source-crs/HardwareEvent.yaml. We only need to add the file in spe.sourceFiles in PolicyGenTemplate to apply it.

Reply:
Thanks for clarifying! I removed these steps.